python/tests/test_fts.py

```diff
@@ -147,3 +147,18 @@ def import_mock(name, *args):
         assert r["id"] == 1
 
     assert rs == rs2
+
+
+def test_null_input(table):
+    table.add(
+        [
+            {
+                "vector": np.random.randn(128),
+                "id": 101,
+                "text": None,
+                "text2": None,
+                "nested": {"text": None},
+            }
+        ]
+    )
```

Comment:
Maybe have one test case that triggers the `doc.is_empty` case and one that does not?